src/con_duct/__main__.py

```diff
@@ -345,46 +367,46 @@ def execution_summary(self) -> dict[str, Any]:
             "exit_code": self.process.returncode,
             "command": self.command,
             "logs_prefix": self.log_paths.prefix,
-            "wall_clock_time": f"{self.elapsed_time:.3f} sec",
+            "wall_clock_time": f"{self.elapsed_time:.3f}",
```

Comment:
why is it still formatted to str here and below -- shouldn't we just leave original numerics here?

Reply:
Seems reasonable to me, this is there just for rounding, otherwise its unhelpfully "precise"

` "wall_clock_time": 1.0147314071655273`

I'll update to use  `round`, which would drop the unnecessary quotes. 

` "wall_clock_time": 1.015`

Reply:
rounding etc should be done by "rendering"/styling function. Is this a rendering function or that is a general record/model composition?

Reply:
addressed in latest push